pom.xml

```diff
@@ -62,6 +61,7 @@
 
     <!-- Required since support for ELS 2.x. Keep in sync with kie-parent or remove when those
           two versions are being updated on the IP BOM.-->
+    <version.org.apache.lucene>6.6.1</version.org.apache.lucene>
```

Comment:
@adrielparedes Because of this version management, there are now two versions of lucene on the classpath for dashbuilder-dataset-cdi. In particular, I cannot build dashbuilder locally because lucene-core:6.6.1 and lucene-misc:5.3.1 are both transitive dependencies, which cause the enforcer to fail because of duplicated classes.

I'm not sure how the dashbuilder PR build passed.